cirq/circuits/frozen_circuit.py

```diff
@@ -176,6 +176,14 @@ def _resolve_parameters_(
     ) -> 'FrozenCircuit':
         return self.unfreeze()._resolve_parameters_(param_resolver, recursive).freeze()
 
+    @functools.wraps(Circuit.tetris_concat)  # Inherit doc string.
```

Comment:
I think using `functools.wraps` overwrites the full signature, so mypy might think this returns `cirq.Circuit` instead of `cirq.FrozenCircuit`. Certainly, that's what happens at runtime:
```python
>>> inspect.signature(cirq.FrozenCircuit.tetris_concat)
<Signature (*circuits: 'cirq.AbstractCircuit', stop_at_first_alignment: bool = False) -> 'cirq.Circuit'>
```
If you just want the docstring, it'd be better to do:
```python
def tetris_concat(...):
    ...

tetris_concat.__doc__ = Circuit.tetris_concat.__doc__
```

Reply:
Or, write an explicit docstring that refers to `Circuit.tetris_concat`.